### PR TITLE
Refine vp-jwt-1.1 production rules

### PR DESCRIPTION
**Pull request by OR13**
***
<a href="https://pr-preview.s3.amazonaws.com/w3c/vc-jwt/pull/48.html" title="Last updated on Feb 15, 2023, 2:07 AM UTC (63db69e)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/vc-jwt/48/aa22ec2...63db69e.html" title="Last updated on Feb 15, 2023, 2:07 AM UTC (63db69e)">Diff</a>

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -646,6 +646,79 @@ <h3>Instead of...</h3>
...
+        We start with an empty header, and payload objects, and we add members
+        to the header and the payload based on the content in the
+        <code>application/presentation+ld+json</code>.
+      </p>
+      <section>
+        <h4>holder -> iss</h4>
+        <p>
+          This member MAY be present in the
+          <code>payload.vp.holder</code> attribute as either a string or an
+          <code>object</code> with and <code>id</code>.
```
Comment (TallTed) [inline index.html:677]:
```suggestion
          <code>object</code> with an <code>id</code>.
```

File: index.html [hunk 2]
```diff
@@ -646,6 +646,79 @@ <h3>Instead of...</h3>
...
+        </p>
+        <p>This member MAY be present <code>payload.iss</code>.</p>
+        <p>
+          In the case that <code>payload.vp.holder</code> is an object,
```
Comment (TallTed) [inline index.html:681]:
```suggestion
          If <code>payload.vp.holder</code> is an object,
```

File: index.html [hunk 3]
```diff
@@ -646,6 +646,79 @@ <h3>Instead of...</h3>
...
+          <code>payload.iss</code> MUST be <code>payload.vp.holder.id</code>.
+        </p>
+        <p>
+          In the case that
```
Comment (TallTed) [inline index.html:685]:
```suggestion
          If
```

File: index.html [hunk 4]
```diff
@@ -646,6 +646,79 @@ <h3>Instead of...</h3>
...
+          <code>payload.vp.holder</code> is a string,
+          <code>payload.iss</code> must be
+          <code>payload.vp.holder</code>
+        </p>
+      </section>
+      <section>
+        <h4>id -> jti</h4>
+        <p>
+          In the case that
```
Comment (TallTed) [inline index.html:694]:
```suggestion
          If
```

**Review by TallTed — COMMENTED** [review]
Less verbose is usually more comprensible...

**Comment by OR13** [comment]
@TallTed thank you for suggestions!

**Review by mprorock — APPROVED** [review]

**Comment by OR13** [comment]
Open for 2 weeks, no objections, merging.

**Pull request merged**